src/apps/vpn/models/servercountrymodel.cpp

```diff
@@ -237,6 +237,18 @@ const QString ServerCountryModel::countryName(
   return QString();
 }
 
+const int ServerCountryModel::indexOfCountryCode(
+    const QString& countryCode) const {
+  int i = 0;
+  for (const ServerCountry& country : m_countries) {
+    if (country.code() == countryCode) {
+      return i;
+    }
+    i++;
+  }
+  return -1;
+}
```

Comment:
Consider using STL for find. Something like:

int ServerCountryModel::indexOfCountryCode(
    const QString& countryCode) const {

    auto it = std::find_if(m_countries.begin(), m_countries.end(),
                           [&countryCode](const Country& country) {
                               return country.code() == countryCode;
                           });

    if (it != m_countries.end()) {
        return std::distance(m_countries.begin(), it);
    } else {
        return -1;
    }
}